README.md

```diff
@@ -65,9 +65,15 @@ but not limited to:
 
 
 ## GETTING STARTED
-* You want to start the RIOT? Just follow our [quickstart guide](http://doc.riot-os.org/index.html#the-quickest-start) or try this [tutorial](https://github.com/RIOT-OS/Tutorials/blob/master/README.md). For specific toolchain installation, follow instructions in the [getting started](http://doc.riot-os.org/getting-started.html) page.
+* You want to start the RIOT? Just follow our
+[quickstart guide](http://doc.riot-os.org/index.html#the-quickest-start) or
+try this
+[tutorial](https://github.com/RIOT-OS/Tutorials/blob/master/README.md).
+For specific toolchain installation, follow instructions in the
```

Comment:
... instructions *on* the ..

Reply:
It was already written like that. My english knowledge is not good enough to know what's best/valid.